vhd/packer/configure-windows-vhd.ps1

```diff
@@ -115,10 +115,9 @@ function Get-FilesToCacheOnVHD
             "https://kubernetesartifacts.azureedge.net/kubernetes/v1.19.0-beta.2/windowszip/v1.19.0-beta.2-1int.zip"
         );
         "c:\akse-cache\win-vnet-cni\" = @(
-            "https://kubernetesartifacts.azureedge.net/azure-cni/v1.1.0/binaries/azure-vnet-cni-windows-amd64-v1.1.0.zip",

```

Comment:
I assume the above binary no longer supports any of the Windows CNI configurations?

cc @jsturtevant

Reply:
sorry i couldn't get it..can you explain a bit more?

Reply:
why are we deleting the `azure-vnet-cni-windows-amd64-v1.1.0.zip` artifact from the Windows VHD configuration?

Reply:
the code keeps only last 3 recent versions right?

Reply:
Just commenting that the package naming convention changed between 1.1.0 and 1.1.2. Just making sure that we have no dependencies upon the old naming convention @ 1.1.0

Reply:
looks like we made the switch to the new naming convention awhile back: 

https://github.com/Azure/aks-engine/blob/efe782ed75913d4760745d49fc1eeedac45b70de/pkg/api/azenvtypes.go#L104

Likely that if someone chose 1.1.0 they would not get the correct package so removing this is probably for the better.